pytorch_lightning/plugins/training_type/deepspeed.py

```diff
@@ -231,6 +231,8 @@ def _init_scheduler_optimizer(self):
         return optimizer, scheduler, optimizer_frequencies
 
     def _initialize_deepspeed_train(self, model):
+        if self.root_device.type == "cuda":
```

Comment:
```suggestion
        if self.on_gpu:
```
Nit: As property is already available.